website/docs/talks/asem-eg-2022-12-16.md

```diff
@@ -0,0 +1,20 @@
+# ASEM Expert Group Digitalisation
+
+As part of the [ASEM Expert Group Digitalisation](https://asem-education.org/priority-areas/digitalisation/) meeting on Friday, December 16, 2022, we gave a talk on Open Education Hub, our vision, goals and work so far.
+
+The ASEM Expert Group Digitalisation meets twice a year to discuss and layout the cooperation between Europe and Asia regarding education, technology and digitalisation.
+Open Education Hub fits well into the agenda and were happy to give a talk and present our work as part of the OpenEdu Project (21-COP-0016), funded by EEA Grants: _Using the Open Source Collaborative Model for the Development, Delivery and Curation of Digital Educational Content_.
+
+The talk focused on Open Education Hub's mission of **using technology to improve all aspects of teaching and learning**.
+
+We started by contrasting the current load of teachers and teaching, filled with administrative and tedious, repetitive tasks, with the ideal environment where teaching (and learning) should be a fun, engaging, human-centered, personalized experience.
+Logistics, management, evaluation and content should be, as much as possible, already in place.
+
+We stated that technology (computing, Internet, devices) provides the means to get many mundane, repetitive things out of the way and to improve teaching / learning via automation and via global connection (for contributions and collaboration).
+Moreover, technology provides accessibility and fairness to (high quality educational) content to those who request / need it.
+
+We gave on overview of our work so far and the tools and techniques we adapted from the open source world: Markdown format, GitHub, [reveal-md](https://github.com/webpro/reveal-md), Docker, GitLab, CI/CD, scripting & automation.
+
+We summarized the challenges we are currently facing, regarding the effort to generate and convert content, making it easy to configure for personal, integration with existing tools and having the manpower to maintain a large set of repositories.
+
+The talk was well received, with questions / recommendations on having the approach available beyond universities.
```

Comment:
Maybe give a few examples of points of feedback that you have received and that we are considering?